services/docker-compose.yml

```diff
@@ -227,6 +229,7 @@ services:
       START_AS_MODE_CPU: ${SIDECAR_FORCE_CPU_NODE:-0}
 
     networks:
+      - default
```

Comment:
ok, so there is no reason to have the sidecar part of the default. Also, it does not need to access the postgres before the director sends computation tasks. so probably you do not need to have that change (and also no wait for from the sidecar).
Maybe you can even fine tune a bit these things?